src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -93,10 +102,31 @@
   &.pf-m-full-height {
     height: 100%;
   }
+
+  &.pf-m-danger {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-danger__icon--Color);
+  }
+
+  &.pf-m-warning {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-warning__icon--Color);
+  }
+
+  &.pf-m-success {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-success__icon--Color);
+  }
+
+  &.pf-m-info {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-info__icon--Color);
+  }
+
+  &.pf-m-custom {
+    --#{$empty-state}__icon--Color: var(--#{$empty-state}--m-custom__icon--Color);
+  }
 }
 
 .#{$empty-state}__content {
   max-width: var(--#{$empty-state}__content--MaxWidth);
+  max-width: var(--#{$empty-state}__content--RowGap);
```

Comment:
did you mean to add this? the width constraints for the size variations isn't working with this in.

Reply:
I did not! I will fix it.